src/command_modules/azure-cli-natgateway/azure/cli/command_modules/natgateway/validators.py

```diff
@@ -0,0 +1,65 @@
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=too-many-lines
+
+from azure.cli.core.commands.client_factory import get_subscription_id
+
+
+def validate_public_ip_addresses(resource):
+    from msrestazure.tools import is_valid_resource_id, resource_id
+
+    def _validate_public_ip_addresses(cmd, namespace):
```

Comment:
I saw a new file above named as `_validators.py`, but contents look different. Can you clarify?